onnxmltools/convert/coreml/operator_converters/neural_network/InnerProduct.py

```diff
@@ -6,32 +6,31 @@
 
 from .....proto import onnx_proto
 from ....common._registration import register_converter
-
+from ....common._apply_operation import apply_reshape
```

Comment:
Swap from `....common._registration import register_converterfrom` and `....common._apply_operation import apply_reshape`